lua/entities/gmod_wire_expression2/core/bone.lua

```diff
@@ -324,7 +324,8 @@ e2function void bone:applyOffsetForce(vector force, vector pos)
 	local ent = isValidBone(this)
 	if not ent then return end
 	if not isOwner(self, ent) then return end
-	if not check(force) or not check(pos) then return end
+	force 	= clamp(force)
+	pos 	= clamp(pos)
```

Comment:
Why was this being checked against max velocity anyways? Odd.. There's [util.IsInWorld](http://wiki.garrysmod.com/page/util/IsInWorld) if that's what was trying to be done..